frontend/src/components/Home.js

```diff
@@ -33,7 +33,22 @@ class Home extends React.Component {
     this.map = null;
   }
 
+  getCensusMBRColor = response_rate => {
+    if (response_rate < 20) {
+      return { color: "#c62828" };
+    } else if (response_rate < 40) {
+      return { color: "#EF6C00" };
+    } else if (response_rate < 60) {
+      return { color: "#FBC02D" };
+    } else if (response_rate < 80) {
+      return { color: "#2E7D32" };
+    } else if (response_rate <= 100) {
+      return { color: "#7CB342" };
+    }
+  };
```

Comment:
@caiyingshen did we agree to make ranges of response rates correlate with different colors? i thought it would be more of a gradient but idk how doable that is

Reply:
This is mainly just a temporary code meant to represent the fact that we will change the text color based on its value. The actual color and separating ranges will probably be changed.

Reply:
Please add a comment for that please!